### PR TITLE
Rename of docs

### PR DESCRIPTION
**Pull request by SwartzCr**
some rename of .rst files

I tried to be smart about just renaming references to the client, but I've also changed some urls to certbot/certbot which will have to wait for the GH repo to be moved

### CODE REVIEW

**Comment by SwartzCr** [comment]
@bmw  wanna take a look over this?

File: docs/api/cb_util.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+:mod:`certbot.cb_util`
```
Comment (bmw) [inline docs/api/cb_util.rst]: This file should be deleted as we didn't rename `le_util` to `cb_util` to prevent breakage.

File: docs/api/le_util.rst [hunk 2]
```diff
@@ -1,5 +0,0 @@
-:mod:`certbot.le_util`
```
Comment (bmw) [inline docs/api/le_util.rst]: Similarly, this file should be added again.

File: docs/ciphers.rst [hunk 3]
```diff
@@ -156,9 +153,9 @@ recommendations with sources of expert guidance on ciphersuites and other
 cryptographic parameters. We're grateful to everyone who contributed
 suggestions. The recommendations we received are available at
 
-https://github.com/letsencrypt/letsencrypt/wiki/Ciphersuite-guidance
+https://github.com/certbot/certbot/wiki/Ciphersuite-guidance
```
Comment (bmw) [inline docs/ciphers.rst]: Do we want to start changing these URLs yet? If we merge this now, our documentation will be broken.

We can either:

1. Wait to merge this PR until we move the repo.
2. Make these changes in a later PR, perhaps even after the 0.6.0 release. Leaving references to `https://github.com/letsencrypt/letsencrypt/*` shouldn't break anything.

I'm personally in favor of 2.

File: docs/contributing.rst [hunk 4]
```diff
@@ -146,7 +146,7 @@ encouraged) make sure to install prerequisites as listed in
 ``certbot-nginx/tests/boulder-integration.sh`` and rerun
 the integration tests suite.
 
-.. _Boulder: https://github.com/letsencrypt/boulder
+.. _Boulder: https://github.com/certbot/boulder
```
Comment (bmw) [inline docs/contributing.rst]: This will remain https://github.com/letsencrypt/boulder

File: docs/using.rst [hunk 5]
```diff
@@ -25,33 +25,33 @@ To install and run the client, just type...
 
 .. code-block:: shell
 
-   ./letsencrypt-auto
+   ./certbot-auto
 
-.. hint:: During the beta phase, Let's Encrypt enforces strict rate limits on
+.. hint:: During the beta phase, Certbot enforces strict rate limits on
```
Comment (bmw) [inline docs/using.rst]: This is referring to the Let's Encrypt server and should not be changed.

File: docs/using.rst [hunk 6]
```diff
@@ -93,10 +93,10 @@ s3front_    Y    Y    Integration with Amazon CloudFront distribution of S3 buck
...
 =========== ==== ==== ===============================================================
 
-.. _plesk: https://github.com/plesk/letsencrypt-plesk
-.. _haproxy: https://code.greenhost.net/open/letsencrypt-haproxy
-.. _s3front: https://github.com/dlapiduz/letsencrypt-s3front
-.. _gandi: https://github.com/Gandi/letsencrypt-gandi
+.. _plesk: https://github.com/plesk/certbot-plesk
+.. _haproxy: https://code.greenhost.net/open/certbot-haproxy
+.. _s3front: https://github.com/dlapiduz/certbot-s3front
+.. _gandi: https://github.com/Gandi/certbot-gandi
```
Comment (bmw) [inline docs/using.rst]: We should not change any of these URLs. We do not know when/if they'll be renamed or what they'll be renamed to.

File: docs/using.rst [hunk 7]
```diff
@@ -130,21 +130,21 @@ specified ``--webroot-path``.  So, for instance,
...
 
 would obtain a single certificate for all of those names, using the
 ``/var/www/example`` webroot directory for the first two, and
 ``/var/www/other`` for the second two.
 
 The webroot plugin works by creating a temporary file for each of your requested
-domains in ``${webroot-path}/.well-known/acme-challenge``. Then the Let's
-Encrypt validation server makes HTTP requests to validate that the DNS for each
-requested domain resolves to the server running letsencrypt. An example request
+domains in ``${webroot-path}/.well-known/acme-challenge``. Then the Certbot
```
Comment (bmw) [inline docs/using.rst]: Reference to Let's Encrypt server

File: docs/using.rst [hunk 8]
```diff
@@ -130,21 +130,21 @@ specified ``--webroot-path``.  So, for instance,
...
+validation server makes HTTP requests to validate that the DNS for each
+requested domain resolves to the server running certbot. An example request
 made to your web server would look like:
 
 ::
 
-    66.133.109.36 - - [05/Jan/2016:20:11:24 -0500] "GET /.well-known/acme-challenge/HGr8U1IeTW4kY_Z6UIyaakzOkyQgPr_7ArlLgtZE8SX HTTP/1.1" 200 87 "-" "Mozilla/5.0 (compatible; Let's Encrypt validation server; +https://www.letsencrypt.org)"
+    66.133.109.36 - - [05/Jan/2016:20:11:24 -0500] "GET /.well-known/acme-challenge/HGr8U1IeTW4kY_Z6UIyaakzOkyQgPr_7ArlLgtZE8SX HTTP/1.1" 200 87 "-" "Mozilla/5.0 (compatible; Certbot validation server; +https://www.certbot.com)"
```
Comment (bmw) [inline docs/using.rst]: Request will still come from Let's Encrypt server.

File: docs/using.rst [hunk 9]
```diff
@@ -272,22 +272,22 @@ you prefer to manage everything by hand, this section provides
...
 your (web) server configuration directly to those files (or create
-symlinks). During the renewal_, ``/etc/letsencrypt/live`` is updated
+symlinks). During the renewal_, ``/etc/certbot/live`` is updated
 with the latest necessary files.
 
-.. note:: ``/etc/letsencrypt/archive`` and ``/etc/letsencrypt/keys``
+.. note:: ``/etc/certbot/archive`` and ``/etc/certbot/keys``
    contain all previous keys and certificates, while
-   ``/etc/letsencrypt/live`` symlinks to the latest versions.
+   ``/etc/certbot/live`` symlinks to the latest versions.
```
Comment (bmw) [inline docs/using.rst]: All `/etc/certbot` paths above are incorrect. I believe we decided not to make this change now (or maybe ever). See #2830.

File: docs/using.rst [hunk 10]
```diff
@@ -340,17 +340,17 @@ Configuration file
...
    :code: ini
 
 By default, the following locations are searched:
 
-- ``/etc/letsencrypt/cli.ini``
-- ``$XDG_CONFIG_HOME/letsencrypt/cli.ini`` (or
-  ``~/.config/letsencrypt/cli.ini`` if ``$XDG_CONFIG_HOME`` is not
+- ``/etc/certbot/cli.ini``
+- ``$XDG_CONFIG_HOME/certbot/cli.ini`` (or
+  ``~/.config/certbot/cli.ini`` if ``$XDG_CONFIG_HOME`` is not
```
Comment (bmw) [inline docs/using.rst]: Path is still `~/.config/letsencrypt/cli.ini`.

File: docs/using.rst [hunk 11]
```diff
@@ -359,21 +359,21 @@ By default, the following locations are searched:
 Getting help
 ============
 
-If you're having problems you can chat with us on `IRC (#letsencrypt @
-Freenode) <https://webchat.freenode.net?channels=%23letsencrypt>`_ or
-get support on our `forums <https://community.letsencrypt.org>`_.
+If you're having problems you can chat with us on `IRC (#certbot @
+OFTC) <https://webchat.oftc.net?channels=%23certbot>`_ or
```
Comment (bmw) [inline docs/using.rst]: Are we moving to OFTC? I'm fine either way, however, @pde and I are hanging out in #certbot on Freenode.

File: docs/using.rst [hunk 12]
```diff
@@ -359,21 +359,21 @@ By default, the following locations are searched:
...
+get support on our `forums <https://community.certbot.org>`_.
```
Comment (bmw) [inline docs/using.rst]: This should still be community.letsencrypt.org.

File: docs/using.rst [hunk 13]
```diff
@@ -359,21 +359,21 @@ By default, the following locations are searched:
...
 tracker
-<https://github.com/letsencrypt/letsencrypt/issues>`_. Remember to
+<https://github.com/certbot/certbot/issues>`_. Remember to
 give us as much information as possible:
 
 - copy and paste exact command line used and the output (though mind
   that the latter might include some personally identifiable
   information, including your email and domains)
-- copy and paste logs from ``/var/log/letsencrypt`` (though mind they
+- copy and paste logs from ``/var/log/certbot`` (though mind they
```
Comment (bmw) [inline docs/using.rst]: Incorrect path

File: docs/using.rst [hunk 14]
```diff
@@ -402,14 +402,14 @@ to, `install Docker`_, then issue the following command:
...
 .. code-block:: shell
 
-   sudo docker run -it --rm -p 443:443 -p 80:80 --name letsencrypt \
-               -v "/etc/letsencrypt:/etc/letsencrypt" \
-               -v "/var/lib/letsencrypt:/var/lib/letsencrypt" \
-               quay.io/letsencrypt/letsencrypt:latest auth
+   sudo docker run -it --rm -p 443:443 -p 80:80 --name certbot \
+               -v "/etc/certbot:/etc/certbot" \
+               -v "/var/lib/certbot:/var/lib/certbot" \
+               quay.io/certbot/certbot:latest auth
```
Comment (bmw) [inline docs/using.rst]: All `-v` arguments are incorrect. Additionally, we should probably keep the quay.io URL as `letsencrypt/letsencrypt`, at least for the time being.

File: docs/using.rst [hunk 15]
```diff
@@ -402,14 +402,14 @@ to, `install Docker`_, then issue the following command:
...
 
 and follow the instructions (note that ``auth`` command is explicitly
 used - no installer plugins involved). Your new cert will be available
-in ``/etc/letsencrypt/live`` on the host.
+in ``/etc/certbot/live`` on the host.
```
Comment (bmw) [inline docs/using.rst]: Bad path

File: docs/using.rst [hunk 16]
```diff
@@ -473,7 +473,7 @@ does include the nginx plugin package:
    emerge -av app-portage/layman
    layman -S
    layman -a mrueg
-   emerge -av app-crypt/letsencrypt-nginx
+   emerge -av app-crypt/certbot-nginx
```
Comment (bmw) [inline docs/using.rst]: Like 3rd party plugins, these instructions should not be changed until we know when/if/how they will be changed.

File: docs/using.rst [hunk 17]
```diff
@@ -10,12 +10,12 @@ User Guide
 Installation
 ============
 
-.. _letsencrypt-auto:
+.. _certbot-auto:
```
Comment (bmw) [inline docs/using.rst]: Similarly, `certbot-auto` doesn't exist yet. We can either wait to merge this or revert these changes for the time being.

**Comment by bmw** [comment]
I did not comment on all GitHub URLs or references to `certbot-auto`. I also found these references to Let's Encrypt which should be changed:
```
ciphers.rst:75:Let's Encrypt project hopes to provide useful defaults that reflect
ciphers.rst:120:The Let's Encrypt project expects to follow the Mozilla recommendations
ciphers.rst:130:The Let's Encrypt project may deviate from the Mozilla recommendations
ciphers.rst:134:team first, before asking the Let's Encrypt project to change
ciphers.rst:139:The Let's Encrypt project will entertain proposals to create a *very*
```

`README.rst` was also not modified. Since #2869 was created, we can potentially wait on this, however, since this file is also used in our readthedocs documentation (see `docs/intro.rst`), this PR should not be merged until the README is updated as well.

Finally, I found references to Let's Encrypt in some of the documentation build scripts. I don't know much about this stuff but maybe they should be changed in well. I found:
```
Makefile:89:    @echo "# qcollectiongenerator $(BUILDDIR)/qthelp/LetsEncrypt.qhcp"
Makefile:91:    @echo "# assistant -collectionFile $(BUILDDIR)/qthelp/LetsEncrypt.qhc"
Makefile:98:    @echo "# mkdir -p $$HOME/.local/share/devhelp/LetsEncrypt"
Makefile:99:    @echo "# ln -s $(BUILDDIR)/devhelp $$HOME/.local/share/devhelp/LetsEncrypt"
conf.py:67:project = u'Let\'s Encrypt'
conf.py:68:copyright = u'2014-2015, Let\'s Encrypt Project'
conf.py:228:htmlhelp_basename = 'LetsEncryptdoc'
conf.py:250:    ('index', 'LetsEncrypt.tex', u'Let\'s Encrypt Documentation',
conf.py:251:     u'Let\'s Encrypt Project', 'manual'),
conf.py:280:    ('index', 'certbot', u'Let\'s Encrypt Documentation',
conf.py:296:    ('index', 'LetsEncrypt', u'Let\'s Encrypt Documentation',
conf.py:297:     u'Let\'s Encrypt Project', 'LetsEncrypt', 'One line description of project.',
make.bat:130:   echo.^> qcollectiongenerator %BUILDDIR%\qthelp\LetsEncrypt.qhcp
make.bat:132:   echo.^> assistant -collectionFile %BUILDDIR%\qthelp\LetsEncrypt.ghc
```

File: docs/using.rst (hunk 11 above)
Comment (SwartzCr) [inline docs/using.rst]: @pde and I are also hanging out in #certbot in OFTC and he mentioned on a call that he preferred OFTC to freenode (for this?)

**Comment by SwartzCr** [comment]
added README.rst, changed all github urls back to LE, but left in references to certbot-auto, so this should be merged only after certbot-auto has been authored

File: README.rst [hunk 18]
```diff
@@ -3,18 +3,18 @@
...
+The Certbot is **BETA SOFTWARE**. It contains plenty of bugs and
 rough edges, and should be tested thoroughly in staging environments before use
 on production systems.
 
 For more information regarding the status of the project, please see
 https://letsencrypt.org. Be sure to checkout the
 `Frequently Asked Questions (FAQ) <https://community.letsencrypt.org/t/frequently-asked-questions-faq/26#topic-title>`_.
 
-About the Let's Encrypt Client
+About Certbot
```
Comment (jsha) [inline README.rst]: Add a second paragraph saying "Until May 2016, Certbot was named simply ``letsencrypt`` or ``letsencrypt-auto``, depending on install method. Instructions on the Internet may still refer to this older name."

File: docs/ciphers.rst [hunk 19]
```diff
@@ -80,30 +78,29 @@ art. However, the Let's Encrypt certificate authority does *not*
...
 
-If you don't use the Let's Encrypt client to configure your server
-directly, because the client doesn't integrate with your server software
-or because you chose not to use this integration, then the cryptographic
-defaults haven't been modified, and the cryptography chosen by the server
-will still be whatever the default for your software was.  For example,
-if you obtain a certificate using *standalone* mode and then manually
-install it in an IMAP or LDAP server, your cryptographic settings will
-not be modified by the client in any way.
+If you don't use Certbot to configure your server directly, because the
```
Comment (jsha) [inline docs/ciphers.rst]: I would recommend against reflowing text, unless there are really egregious line length issues, since it makes `git blame` less useful.

File: docs/contributing.rst [hunk 20]
```diff
@@ -175,8 +175,8 @@ Configurators may implement just one of those).
 There are also `~certbot.interfaces.IDisplay` plugins,
 which implement bindings to alternative UI libraries.
 
-.. _interfaces.py: https://github.com/letsencrypt/letsencrypt/blob/master/certbot/interfaces.py
-.. _plugins/common.py: https://github.com/letsencrypt/letsencrypt/blob/master/certbot/plugins/common.py#L34
+.. _interfaces.py: https://github.com/letsencrypt/letsencrypt/blob/master/letsencrypt/interfaces.py
```
Comment (jsha) [inline docs/contributing.rst]: This change is incorrect.

File: docs/contributing.rst [hunk 21]
```diff
@@ -175,8 +175,8 @@ Configurators may implement just one of those).
...
+.. _plugins/common.py: https://github.com/letsencrypt/letsencrypt/blob/master/letsencrypt/plugins/common.py#L34
```
Comment (jsha) [inline docs/contributing.rst]: This one too.

File: docs/packaging.rst [hunk 22]
```diff
@@ -3,4 +3,4 @@ Packaging Guide
 ===============
 
 Documentation can be found at
-https://github.com/letsencrypt/letsencrypt/wiki/Packaging.
+https://github.com/certbot/certbot/wiki/Packaging.
```
Comment (jsha) [inline docs/packaging.rst]: This change is premature until the repo rename.

File: docs/using.rst [hunk 23]
```diff
@@ -25,33 +25,33 @@ To install and run the client, just type...
 
 .. code-block:: shell
 
-   ./letsencrypt-auto
+   ./certbot-auto
 
-.. hint:: During the beta phase, Let's Encrypt enforces strict rate limits on
-   the number of certificates issued for one domain. It is recommended to
-   initially use the test server via `--test-cert` until you get the desired
+.. hint:: During the beta phase, the Let's Encrypt servers enforce strict rate
```
Comment (jsha) [inline docs/using.rst]: While we're in here, can we remove "During the beta phase," and "strict?"

File: docs/using.rst [hunk 24]
```diff
@@ -130,21 +130,21 @@ specified ``--webroot-path``.  So, for instance,
...
-requested domain resolves to the server running letsencrypt. An example request
+domains in ``${webroot-path}/.well-known/acme-challenge``. Then the Let's Encrypt
+validation server makes HTTP requests to validate that the DNS for each
+requested domain resolves to the server running certbot. An example request
 made to your web server would look like:
 
 ::
 
-    66.133.109.36 - - [05/Jan/2016:20:11:24 -0500] "GET /.well-known/acme-challenge/HGr8U1IeTW4kY_Z6UIyaakzOkyQgPr_7ArlLgtZE8SX HTTP/1.1" 200 87 "-" "Mozilla/5.0 (compatible; Let's Encrypt validation server; +https://www.letsencrypt.org)"
+    66.133.109.36 - - [05/Jan/2016:20:11:24 -0500] "GET /.well-known/acme-challenge/HGr8U1IeTW4kY_Z6UIyaakzOkyQgPr_7ArlLgtZE8SX HTTP/1.1" 200 87 "-" "Mozilla/5.0 (compatible; Let's Encryptvalidation server; +https://www.letsencrypt.org)"
```
Comment (jsha) [inline docs/using.rst]: Missing space.

File: docs/using.rst [hunk 25]
```diff
@@ -359,8 +359,8 @@ By default, the following locations are searched:
 Getting help
 ============
 
-If you're having problems you can chat with us on `IRC (#letsencrypt @
-Freenode) <https://webchat.freenode.net?channels=%23letsencrypt>`_ or
+If you're having problems you can chat with us on `IRC (#certbot @
```
Comment (jsha) [inline docs/using.rst]: I think we're keeping #letsencrypt on Freenode for now.

File: docs/using.rst [hunk 26]
```diff
@@ -420,7 +420,7 @@ Operating System Packages
 
 **FreeBSD**
 
-  * Port: ``cd /usr/ports/security/py-letsencrypt && make install clean``
+  * Port: ``cd /usr/ports/security/py-letsencrypt make install clean``
```
Comment (jsha) [inline docs/using.rst]: Missing &&

**Comment by jsha** [comment]
LGTM

File: docs/ciphers.rst [hunk 27]
```diff
@@ -127,15 +124,15 @@ which uses the ChaCha and Poly1305 algorithms, and which is already
 implemented by the Chrome browser.  Mozilla has delayed recommending
 ``0xcc13`` over compatibility and standardization concerns, but is likely
 to recommend it in the future once these concerns have been addressed. At
-that point, the Let's Encrypt client would likely follow the Mozilla
-recommendations and favor the use of this ciphersuite as well.)
+that point, Certbot would likely follow the Mozilla recommendations and favor
+the use of this ciphersuite as well.)
 
 The Let's Encrypt project may deviate from the Mozilla recommendations
```
Comment (SwartzCr) [inline docs/ciphers.rst]: @jsha what do you think about the use of LEt's Encrypt in this paragraph and the next?

Comment (jsha) [inline docs/ciphers.rst]: You mean this paragraph and the previous? I think Certbot is more accurate here.

File: certbot-compatibility-test/docs/conf.py [hunk 28]
```diff
@@ -311,7 +311,7 @@
 intersphinx_mapping = {
     'python': ('https://docs.python.org/', None),
     'acme': ('https://acme-python.readthedocs.org/en/latest/', None),
-    'certbot': ('https://letsencrypt.readthedocs.org/en/latest/', None),
+    'certbot': ('https://certbot.eff.org/docs/', None),
     'certbot-apache': (
         'https://letsencrypt-apache.readthedocs.org/en/latest/', None),
```
Comment (schoen) [inline certbot-compatibility-test/docs/conf.py]: How come this one didn't change?

File: docs/ciphers.rst [hunk 29]
```diff
@@ -127,15 +124,15 @@ which uses the ChaCha and Poly1305 algorithms, and which is already
...
 in the future if good cause is shown and we believe our users'
 priorities would be well-served by doing so. In general, please address
 relevant proposals for changing priorities to the Mozilla security
-team first, before asking the Let's Encrypt project to change the
-client's priorities. The Mozilla security team is likely to have more
+team first, before asking the Let's Encrypt project or EFF to change
```
Comment (schoen) [inline docs/ciphers.rst]: This should just be "the Certbot developers" (rather than "the Let's Encrypt project or EFF").

File: docs/ciphers.rst [hunk 30]
```diff
@@ -172,23 +169,22 @@ This will probably look something like
...
-or
+to enable updating ciphers with each new Certbot release, or
 
 .. code-block:: shell
 
-  letsencrypt --update-ciphers off
+  certbot --update-ciphers off
 
 to disable automatic configuration updates. These features have not yet
 been implemented and this syntax may change then they are implemented.
```
Comment (schoen) [inline docs/ciphers.rst]: Should be "when they are implemented".

File: docs/using.rst [hunk 31]
```diff
@@ -525,19 +530,19 @@ whole process is described in the :doc:`contributing`.
...
 
 Comparison of different methods
 -------------------------------
 
-Unless you have a very specific requirements, we kindly ask you to use
-the letsencrypt-auto_ method. It's the fastest, the most thoroughly
+Unless you have a very specific requirements, we kindly suggest that you use
+the certbot-auto_ method. It's the fastest, the most thoroughly
 tested and the most reliable way of getting our software and the free
 SSL certificates!
```
Comment (schoen) [inline docs/using.rst]: Should be "TLS/SSL certificates", "TLS certificates", or "certificates".

**Comment by schoen** [comment]
I just added four comments and am otherwise fine with this.

**Comment by schoen** [comment]
LGTM, still curious about the `letsencrypt-apache` reference (the documentation still says that the plugin will have the old name) -- is that a renaming-sequence problem?

**Comment by schoen** [comment]
LGTM :-)

**Comment by bmw** [comment]
LGTM
